types/jquery/JQuery.d.ts

```diff
@@ -83,6 +83,7 @@ $( document.body )
      * @since 1.4
      */
     add(selector: JQuery.Selector, context: Element): this;
+    // TODO: The return type should reflect newly selected types.
```

Comment:
I am not sure I really understand this TODO comment. How can we represent the element types after added?